default/stringtables/en.txt

```diff
@@ -4352,11 +4352,25 @@ RESEARCH_TITLE
 Research
 
 RESEARCH_TEXT
-'''The Research (or RP "Research Points") of all planets in the empire is combined to discover new technologies. Any Research that cannot be spent on a tech is lost each turn. Each tech has a minimum number of turns required to research it. For example, if a tech that cost 15 RPs, and had a minimum time of 3 turns, 5 RPs is the maximum that could be put towards its completion per turn.
+'''The Research (or RP "Research Points") of all planets in the empire is combined to discover new [[encyclopedia TECHNOLOGIES_TITLE]].
+
+Any Research that cannot be spent on a tech is lost each turn. Each tech has a minimum number of turns required to research it. For example, if a tech that cost 15 RPs, and had a minimum time of 3 turns, 5 RPs is the maximum that could be put towards its completion per turn.
 The tech at the top of the queue is given RPs first. Any remaining RP are then given to the next tech in the queue, and if there is still more remaining to the next-- and so on. You can re-prioritize the items in the queue by dragging the most important items closer to the top.
 
 Research does not require [[encyclopedia SUPPLY_TITLE]] line connections to be used.'''
 
+TECHNOLOGIES_TITLE
+Technologies
+
+TECHNOLOGIES_TEXT
+'''New attributes, improved planetary stats, buildings, and ship parts can be unlocked by researching the associated technologies.
```

Comment:
I didn't know how to name it... For example, it's the 10% repair bonus granted by techs like Advanced Damage Control.